src/btree/bt_curnext.c

```diff
@@ -851,15 +854,16 @@ __wt_btcur_next_prefix(WT_CURSOR_BTREE *cbt, WT_ITEM *prefix, bool truncating)
             LF_SET(WT_READ_VISIBLE_ALL);
 
         /*
-         * If we are running with snapshot isolation, and not interested in returning tombstones, we
-         * could potentially skip pages. The skip function looks at the aggregated timestamp
-         * information to determine if something is visible on the page. If nothing is, the page is
-         * skipped.
+         * If we are running with snapshot isolation, have a snapshot, and are not interested in
```

Comment:
Looks like this comment was missed on 7.0

Reply:
It is the only change required in 6.0 due to the checkpoint cursors that are not supported, but there exists so many tests that test checkpoint cursors.